src/project-css/app/Http/Controllers/WizardController.php

```diff
@@ -0,0 +1,294 @@
+<?php
+/**
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Http\Controllers;
+
+use App\Models\Collection;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Storage;
+use App\Helpers\TableHelper;
+use Auth;
+
+class WizardController extends Controller
+{
+    private $strDir;
+    private $sharedViewData;
+
+    public function __construct($storageFolder = 'flatfiles') {
+      // Protection to make sure this is only accessible to admin
+      $this->middleware('admin');
+
+      // Storage directory
+      $this->strDir = $storageFolder;
+
+      // Shared View Data is used by both the flatfile and cms
+      // functions to correctly render the file import views.
+      $this->sharedViewData = [
+            'AuthUsr' => Auth::user(),
+            'collcntNms' => Collection::all(),
+            'fltFleList' => $this->flatFileList(),
+            'colID' => 1
+       ];
+    } 
+
+   /**
+    * Returns view for the Import Wizard Main Page
+    *
+    * @return view
+    */    
+    public function import() {
+        return view('admin/wizard/import')->with('AuthUsr', Auth::user());
+    }
+
+    public function importCollection($colID) {
```

Comment:
Needs Proper Documentation: 
Use full PHP Documentation Standards.
Be clear on what the parameters of the function are as well as what the return values are supposed to be.